File: API/PageAndRDL/ReportWizard/C#/ReportWizard/MetaData/ReportMetaData.cs
using System.Collections.Generic;

namespace GrapeCity.ActiveReports.Samples.ReportWizard.MetaData
{
	public class ReportMetaData
	{
		private string _title;

		public ReportMetaData()
		{
			Title = "";
			MasterReportFile = "";
			Fields = new Dictionary<string, FieldMetaData>();
		}

		// property needed for data binding purposes
		public string Title
		{
			get { return _title; }
			set { _title = value; }
		}

		public string MasterReportFile;
		public readonly Dictionary<string, FieldMetaData> Fields;
	}
}

File: API/PageAndRDL/ReportWizard/VB.NET/ReportWizard/MetaData/ReportMetaData.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Namespace MetaData
	Public Class ReportMetaData
		Private _title As String
		Public Sub New()
			Title = ""
			MasterReportFile = ""
			Fields = New Dictionary(Of String, FieldMetaData)()
		End Sub
		' property needed for data binding purposes
		Public Property Title() As String
			Get
				Return _title
			End Get
			Set(ByVal value As String)
				_title = value
			End Set
		End Property

		Public MasterReportFile As String
		Public ReadOnly Fields As Dictionary(Of String, FieldMetaData)
	End Class
End Namespace
